a3fe/configuration/engine_config.py

```diff
@@ -5,298 +5,243 @@
+                    0.247, 0.286, 0.329, 0.373, 0.417, 0.467, 0.514,
+                    0.564, 0.623, 0.696, 0.833, 1.0,
+                ],
+            },
+            "FREE": {
+                "DISCHARGE": [0.0, 0.222, 0.447, 0.713, 1.0],
+                "VANISH": [
+                    0.0, 0.026, 0.055, 0.09, 0.126, 0.164, 0.202, 0.239,
+                    0.276, 0.314, 0.354, 0.396, 0.437, 0.478, 0.518,
+                    0.559, 0.606, 0.668, 0.762, 1.0,
+                ],
+            },
+        },
+        description="Default lambda values for each leg and stage type"
+    )
     ### Integrator - ncycles modified as required by a3fe ###
     nmoves: int = _Field(25000, description="Number of moves per cycle")
     timestep: float = _Field(4.0, description="Timestep in femtoseconds(fs)")
-    runtime: _Union[int, float] = _Field(..., description="Runtime in nanoseconds(ns)")
+    runtime: _Union[int, float] = _Field(5.0, description="Runtime in nanoseconds(ns)")
 
-    input_dir: str = _Field(..., description="Input directory containing simulation config files")
-    @staticmethod
-    def _calculate_ncycles(runtime: float, time_per_cycle: float) -> int:
-        """
-        Calculate the number of cycles given a runtime and time per cycle.
+    ### Constraints ###
+    constraint: str = _Field("hbonds", description="Constraint type, must be hbonds or allbonds")
+    hydrogen_mass_factor: float = _Field(3.0, alias="hydrogen mass repartitioning factor", description="Hydrogen mass repartitioning factor")
+    integrator: _Literal["langevinmiddle", "leapfrogverlet"] = _Field("langevinmiddle", description="Integration algorithm")
+    
+    ### Thermostatting already handled by langevin integrator
+    thermostat: bool = _Field(False, description="Enable thermostat")
+    inverse_friction: float = _Field(1.0, description="Inverse friction in picoseconds", alias="inverse friction")
+    temperature: float = _Field(25.0, description="Temperature in Celsius")
 
-        Parameters
-        ----------
-        runtime : float
-            Runtime in nanoseconds.
-        time_per_cycle : float
-            Time per cycle in nanoseconds.
-
-        Returns
-        -------
-        int
-            Number of cycles.
-        """
-        return round(runtime / time_per_cycle)
+    ### Barostat ###
+    barostat: bool = _Field(True, description="Enable barostat")
+    pressure: float = _Field(1.0, description="Pressure in atm")
 
-    def _set_n_cycles(self, n_cycles: int) -> None:
-        """
-        Set the number of cycles in the SOMD config file.
+    ### Non-Bonded Interactions ###
+    cutoff_type: _Literal["cutoffperiodic", "PME"] = _Field("cutoffperiodic", description="Type of cutoff to use. Options: PME, cutoffperiodic")
+    cutoff_distance: float = _Field(12.0, alias="cutoff distance", ge=6.0, le=18.0, description="Cutoff distance in angstroms (6-18). Default 12.0 for cutoffperiodic and 10.0 for PME")
```

Comment:
We probably just want one default, otherwise we're going to need some more complex logic and may end up overwriting user input. Let's stick with the RF defaults of RF and 12 A.